### DISCUSSION TITLE
Need a rule to require specific PR labels before merging

### DISCUSSION THREAD

**Discussion by gtjoseph** [community/community#129731] [Repositories]
### Select Topic Area

Product Feedback

### Body

While the "Require status checks to pass" rule works, it can be difficult to use because it depends on specific workflow names.  This is an issue if workflow names change or if only one of several workflows has to pass to allow merging.  A rule that requires a specific label would make things a lot easier because it wouldn't be dependent on the workflow name, multiple workflows could set the label, and the label could be set or removed by someone with appropriate permissions allowing an override capability.

**Comment by patrick-knight** [comment]
Would a label be more helpful or wildcard patterns similar to how branch targeting works? 

In your work could you talk a bit more about who can change rules vs who can change workflows? And would love to learn more about how frequently workflow names change?
